.github/workflows/tests.yml

```diff
@@ -44,6 +45,24 @@ jobs:
 
       - run: brew typecheck
 
+  update-test:
+    name: update test
+    needs: syntax
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-22.04
```

Comment:
Given the critical nature of this, it might be worth continuing to test this on macOS too.

Reply:
@Bo98 happy to do so. just don't think I've ever seen it only fail on one platform?

Reply:
In the end we can break the rest of Homebrew all we like: if `brew update` still works then we can sort things out.

If `brew update` breaks then we're in a difficult spot.

Reply:
@Bo98 cool, added macOS job.